### ISSUE TITLE
Issue with cache and PFC

### ISSUE THREAD

**Issue by meholidays** [chapagain/auto-currency-switcher#5]
First let me thank you for this free extension that works great in setting the currency automatically. There is however an issue that I hope can be addressed. The extension seems not to work properly in Magento 1.6.2 with the block cache active. Also does not work with Lesti_FPC active. What happens is that the first country that hits the website, sets the currency for all customers, regardless of their locations. Anybody has a suggestion on how this can be corrected? I have cache disabled because of this extension. I have a feeling this has something to do with session and cookies.

**Comment by zakkimughal** [comment]
yes mine is the same problem this is not friendly with cache I can confirm. It works great if you have no cache
